src/TOMLInterface.jl

```diff
@@ -243,6 +247,13 @@ function get_distribution_from_expr(d::Expr)
 
         return dist_type(dist_args)
 
+    elseif dist_type_symb == Symbol("constrained_gaussian")
```

Comment:
There can also be a `repeats=N` optional argument that should be passed through too,

Reply:
Do we need full kwarg functionality or just `repeats=N`?